core/src/main/java/io/jenkins/pluginhealth/scoring/model/ProbeResult.java

```diff
@@ -27,41 +27,47 @@
 import java.time.ZonedDateTime;
 import java.util.Objects;
 
+import com.fasterxml.jackson.annotation.JsonAlias;
+
 /**
  * Represents the result of one analyze performed by a {@link io.jenkins.pluginhealth.scoring.probes.Probe} implementation on a {@link Plugin}
  *
  * @param id      represent the ID of the {@link io.jenkins.pluginhealth.scoring.probes.Probe}
  * @param message represents a summary of the result
  * @param status  represents the state of the analyze performed
  */
-public record ProbeResult(String id, String message, ResultStatus status, ZonedDateTime timestamp) {
-    public ProbeResult(String id, String message, ResultStatus status) {
-        this(id, message, status, ZonedDateTime.now());
+public record ProbeResult(String id, String message, Status status, ZonedDateTime timestamp, long probeVersion) {
```

Comment:
Introduction of a version for probe let re-run the probes when we change them (bug fix, improvements) even if they should normally not be retriggered (no changes in the plugin, or no new releases of the plugin)

Reply:
Done, thanks for the reminder.